doc/topics/the-event-loop-timers-and-nexttick.md

```diff
@@ -131,78 +131,77 @@ someAsyncOperation(function () {
+When the event loop enters the **poll** phase _and there are no timers
 scheduled_, one of two things will happen:
 
-* _If the `poll` queue **is not empty**_, the event loop will iterate
-through  its queue of callbacks executing them synchronously until
-either the queue has  been exhausted, or the system-dependent hard limit
+* _If the **poll** queue **is not empty**_, the event loop will iterate
+through its queue of callbacks executing them synchronously until
+either the queue has been exhausted, or the system-dependent hard limit
 is reached.
 
-* _If the `poll` queue **is empty**_, one of two more things will
+* _If the **poll** queue **is empty**_, one of two more things will
 happen:
   * If scripts have been scheduled by `setImmediate()`, the event loop
-  will end the `poll` phase and continue to the `check` phase to
+  will end the **poll** phase and continue to the **check** phase to
   execute those scheduled scripts.
 
   * If scripts **have not** been scheduled by `setImmediate()`, the
   event loop will wait for callbacks to be added to the queue, then
-  execute it immediately.
+  execute them immediately.
 
-Once the `poll` queue is empty the event loop will check for timers
+Once the **poll** queue is empty the event loop will check for timers
 _whose time thresholds have been reached_. If one or more timers are
-ready, the event loop will wrap back to the timers phase to execute
+ready, the event loop will wrap back to the **timers** phase to execute
 those timers' callbacks.
 
-### `check`:
+### **check**
 
-This phase allows a person to execute callbacks  immediately after the
-`poll` phase has completed. If the `poll` phase  becomes idle and
-scripts have been queued with `setImmediate()`, the   event loop may
-continue to the `check` phase rather than waiting.
+This phase allows a person to execute callbacks immediately after the
+**poll** phase has completed. If the **poll** phase becomes idle and
+scripts have been queued with `setImmediate()`, the event loop may
+continue to the **check** phase rather than waiting.
 
 `setImmediate()` is actually a special timer that runs in a separate
 phase of the event loop. It uses a libuv API that schedules callbacks to
-execute after the `poll` phase has completed.
+execute after the **poll** phase has completed.
 
 Generally, as the code is executed, the event loop will eventually hit
-the `poll` phase where it will wait for an incoming connection, request,
-etc. However, after a callback has been scheduled with `setImmediate()`,
-then the  `poll` phase becomes idle, it will end and continue to the
-`check`  phase rather than waiting for `poll` events.
+the **poll** phase where it will wait for an incoming connection, request,
+etc. However, if a callback has been scheduled with `setImmediate()`
+and the **poll** phase becomes idle, it will end and continue to the
+**check** phase rather than waiting for **poll** events.
 
-### `close callbacks`:
+### **close callbacks**
```

Comment:
@cjihrig Only to follow the pattern of bolding event loop phases. Visually, I think it looked better bolded than not, but that is just me personally.